### ISSUE TITLE
[tf.keras 1.8.0] the size of loaded model increases after calling model.fit()

### ISSUE THREAD

**Issue by HanSangJun** [tensorflow/tensorflow#25308]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow version (use command below)**: 1.8.0
- **Python version**: 3.5.2
- **GPU model and memory**: GTX 1080 Ti

### Describe the problem
I'm using tf.keras. I'm working with Densenet169 from tensorflow.python.keras.applications.densenet. I checked the model size (almost 53 MB) on Keras website, and obviously the size could be verified by saving it directly (using save_model()) on my PC before training. When calling mode.fit(), I found the size of model increased up to 153 MB without any changes of the number of parameter, data type and model structure (even for 1 epoch). Do you have any idea?

### The list of what I tested
- Comparison model.get_config() before and after training -> no difference
- The possibility that a list variable for validation set was not set free in model.fit() function -> no difference whether validation set exists or not
- Data type for the weights -> fixed as float32
- Callback functions (checkpoint and earlystopping) -> no effect
- model.compile() was called for several times to drop learning rate after a few epochs -> no relationship between model.compile() and model size

**Comment by jvishnuvardhan** [comment]
Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. Github is mainly for addressing bugs in installation and performance. Thanks!

**Comment by jvishnuvardhan** [comment]
I am closing this as it is not related to Build/Installation or Bug/Performance. Thanks!
